WorkspaceServer/Kernel/CSharpKernel.cs

```diff
@@ -66,7 +66,8 @@ private void SetupScriptOptions()
                     typeof(Task<>).Assembly,
                     typeof(IKernel).Assembly, 
                     typeof(CSharpKernel).Assembly, 
-                    typeof(PocketView).Assembly);
+                    typeof(PocketView).Assembly,
+                    typeof(XPlot.Plotly.PlotlyChart).Assembly);
```

Comment:
Do we really need this reference by default? What if we left it up to the user to `#r "nuget:XPlot"` if they want?

Reply:
We can go either way. Initially though, since the extension is referenced by the tool at build time, and we haven't improved the assembly loader, I'm concerned that `#r nuget` will run us afoul of the multiple assembly load bug.

Reply:
Ok, that seems reasonable to me, for now.